code/game/objects/items/storage/book.dm

```diff
@@ -115,30 +125,19 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 	if(user.mind && (user.mind.isholy))
 		chaplain = 1
 
-	if(!chaplain)
```

Comment:
leave it to chaplains, not everyone in the wasteland is holy

Reply:
still needs a chaplain to use, but it doesn't burn you alive if you're *not* a chaplain now